### ISSUE TITLE
Is it normal that the headset produces data without putting it on the head?

### ISSUE THREAD

**Issue by hungluchan** [trentbrooks/BrainWaveOSC#14]
Thanks for the BrainWaveOSC! It works very well with the MindWave Mobile2. Just wondering if it is normal that the headset is still producing data without putting the headset on my head?

**Comment by thedavester1** [comment]
Ive been using it too! It works great with that brain wave sensor, I use it too. I got data when the headset isn't on either. the quality of that signal will be really bad which should be seen in BrainWaveOSC and those data values are probably feedback from the device and air but unsure thats what I think

**Comment by hungluchan** [comment]
Right. I was also thinking if the weird data values are the feedback from the air or something... By the way, do you also get a very high percentage of Delta and a very low percentage of Gamma? Not sure if the values are placed reversely? Otherwise, I thought the Delta should be lowest when awake...?
